Gemfile.local

```diff
@@ -1,5 +1,3 @@
-source 'https://rubygems.org'
-
-# each plugin's native extensions should be added to here
-gem "rgeo"
-gem "pg", (ENV['GEM_PG_VERSION'] ? "~> #{ENV['GEM_PG_VERSION']}" : "~> 1.2.3")
+gem 'puma'
+gem 'puma_worker_killer'
+gem 'lograge'
```

Comment:
@dkastl (CC: @smellman)
In my opinion, 'puma_worker_killer` is for production environment, but shall we assume this `docker-gtt` setup for `production` environment (not for `development` environment) ?

Also, `lograge` is for changing rails log as JSON format which is convenience on AWS ECS log and CloudWatch, but shall we assume AWS as first deployment priority ?

Reply:
@dkastl  
> but we actually run it --without test development and with production environment,

Okay, sure at this moment...
By the way, I found the way to support both (`production` and `test development`) env in another project, so it can be solved.

For debugging purpose, multiple workers (=3) and `puma_worker_killer` is quite annoying for me, so personally configurable this part with additional environment variables will be the best way.

> Also, `lograge` is for changing rails log as JSON format which is convenience on AWS ECS log and CloudWatch, but shall we assume AWS as first deployment priority ?

How about this ?

Reply:
By the way, `mailcatcher` is used in `docker-compose.yml`, so it's a kind of half `production` and half `development` env.

Reply:
> Well, don't know very much about these details. Could you just make a change that works good for development and push it to this branch?

Do you mean only the following part ?

> By the way, I found the way to support both (production and test development) env in another project, so it can be solved.

If so, it's okay for me.

But if it's also gem dependencies (`puma`, `puma_worker_killer` and `lograge`), we need to decide something project scope ,etc ?